elephant/functional_connectivity.py

```diff
@@ -0,0 +1,6 @@
+from elephant.functional_connectivity_src.total_spiking_probability_edges import (
+    total_spiking_probability_edges,
+    get_connectivity_matrix,
```

Comment:
Maybe you could consider to only make `total_spiking_probability_edges` function be made available to the user, rather than exposing the internal `get_connectivity_matrix` function. This would simplify the module's API for the user and make it easier to add more functional connectivity measures in the future without affecting the user-facing interface.

As for the output of `total_spiking_probability_edges`, it may return both the connectivity matrix and delay matrix directly, depending on how the function is implemented ? would this be sensible ?